portfolio/src/main/webapp/intro.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sarah Zou's STEP  Portfolio</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <!--<link rel="stylesheet" href="style.css">-->
+    <script src="script.js"></script>
+  </head>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="#">Sarah Zou</a> <!-- NEED TO FIX COLLAPSE NAV BAR-->
```

Comment:
Is the navbar currently broken? Or is this something that needs to be tested / implemented later?